driver/src/orderbook/mod.rs

```diff
@@ -8,6 +8,7 @@ use self::auction_data_reader::PaginatedAuctionDataReader;
 pub use self::filtered_orderbook::{FilteredOrderbookReader, OrderbookFilter};
 pub use self::onchain_filtered_orderbook::OnchainFilteredOrderBookReader;
 pub use self::shadow_orderbook::ShadowedOrderbookReader;
+pub use self::streamed::Orderbook as EventOrderbook;
```

Comment:
naming nit: This sounds a bit like an orderbook full of events to me. Should we call it EventBasedOrderook or something like that (maybe there is a better option than that).

Reply:
I like `EventBasedOrderbook` but could also get into `EventStreamedOrderbook` if anyone is interested.